src/services/suggestUsersService.js

```diff
@@ -0,0 +1,12 @@
+const suggestedList = (users, searchVal, limit) => {
+    return users
+        .filter((user) => {
+            return user.login.includes(searchVal);
```

Comment:
@lenon777, remove `return` keyword. The arrow function was designed to be single-lined.